src/brokers/view-brokers/components/PreConfirmDeleteModal/PreConfirmDeleteModal.tsx

```diff
@@ -0,0 +1,42 @@
+import { FC, Fragment } from 'react';
+import { Modal, ModalVariant, Button } from '@patternfly/react-core';
+import { useTranslation } from '../../../../i18n';
+import { Trans } from 'react-i18next';
+
+interface OnDeleteClickProps {
+  isModalOpen: boolean;
+  onDeleteButtonClick: () => void;
+  handleModalToggle: () => void;
+}
+
+const PreConfirmDeleteModal: FC<OnDeleteClickProps> = ({
+  onDeleteButtonClick,
+  isModalOpen,
+  handleModalToggle,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <Fragment>
```

Comment:
```suggestion
    
```
We don't need fragment here. Fragment require when we have more than one sibling tag